migrations/1660716115917-seedNotificationType.ts

```diff
@@ -4,7 +4,10 @@ import {
   SCHEMA_VALIDATORS_NAMES,
 } from '../src/entities/notificationType';
 import { MICRO_SERVICES, THIRD_PARTY_EMAIL_SERVICES } from '../src/utils/utils';
+import { SegmentEvents } from '../src/services/segment/analytics';
+import { NOTIFICATION_CATEGORY } from '../src/types/general';
 
+// https://github.com/Giveth/notification-center/issues/6 , https://gist.github.com/MohammadPCh/24434d50bc9ccd9b74905c271ee05482
```

Comment:
We are missing the welcome notification type. I think we removed it on another PR. We should re add it, since it's part of the logic.